core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -811,9 +810,16 @@ public String toString(@Nullable AbstractBlockChain chain, @Nullable Network net
                         s.append('\n');
                     }
                 } catch (Exception e) {
-                    s.append("[exception: ").append(e.getMessage()).append("]\n");
+                    s.append(ByteUtils.formatHex(getInput(0).getScriptBytes()));
+                    if (!isCoinBase()) {
```

Comment:
Shouldn't we be using `in` instead of `getInput(0)` here?

Reply:
Absolutely! Thanks for spotting this.